sdk/keyvault/Azure.Security.KeyVault.Shared/src/ChallengeBasedAuthenticationPolicy.cs

```diff
@@ -176,6 +197,11 @@ internal ChallengeParameters(Uri authorizationUri, string[] scopes)
             /// Gets the tenant ID from <see cref="AuthorizationUri"/>.
             /// </summary>
             public string TenantId { get; }
+
+            /// <summary>
+            /// Gets the "claims" parameter from the challenge response if Continuous Access Evaluation is enabled.
+            /// </summary>
+            public string Claims { get; set; }
```

Comment:
Strange that this member is read/write when the others are read-only.  Any way that we can do the conditional population logic in the constructor?   Otherwise, it may be better to have an `UpdateClaims` method and preserve the R/O property accessors.